### PR TITLE
Evan buzzfeed quiz edits

### PR DESCRIPTION
**Pull request by evanzhong**
Bunch of changes in preparation for project walkthrough with the kids.

Main focuses
- Encapsulate as much jQuery code as possible into helper functions with appropriate (English readable) function signatures
  - Part of the lesson will focus on the power of modularization and encapsulation as a tool to reduce complexity
- Descriptive (English readable) variable names
- Reduce the number of lines in `script.js` to make reading it less overwhelming for students
- Refresh / Introduce dictionaries

Minor focuses
- Whitespace and formatting changes
- Readiness for codepen

### CODE REVIEW

File: buzzfeed-style-quiz/src/index.html [hunk 1]
```diff
@@ -53,4 +51,5 @@ <h2 class="quiz-question">Q6:</h2>
 
 <div class="quiz-result"></div>
 
-
+<!-- Magic things that make the quiz work behind the scenes! -->
+<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
```
Comment (evanzhong) [inline buzzfeed-style-quiz/src/index.html:55]: We need to include this so when students copy our codepen the appropriate jQuery libraries are linked

File: buzzfeed-style-quiz/src/script.js [hunk 2]
```diff
@@ -1,102 +1,126 @@
...
-  5:  {'a':1,'b':2,'c':3,'d':4},
-  6:  {'a':1,'b':2,'c':3,'d':4}
+let questions = {
+  //This should be changed!
+  'Q1':  {'a':1,'b':2,'c':3,'d':4},
+  'Q2':  {'a':2,'b':3,'c':4,'d':1},
+  'Q3':  {'a':2,'b':4,'c':1,'d':3},
+  'Q4':  {'a':3,'b':1,'c':2,'d':4},
+  'Q5':  {'a':1,'b':2,'c':3,'d':4},
+  'Q6':  {'a':1,'b':2,'c':3,'d':4}
```
Comment (evanzhong) [inline buzzfeed-style-quiz/src/script.js:10]: I opted to change the top level keys here to strings `1` => `Q1`

It reduces some ambiguity and thus makes describing the different levels in our nested dictionary a little easier.

File: buzzfeed-style-quiz/src/script.js [hunk 3]
```diff
@@ -1,102 +1,126 @@
...
-
-
 }
 
 
 let Quiz = function(){
-  let self = this;
+  let myQuiz = this;
   this.init = function(){
-    self._bindEvents();
```
Comment (evanzhong) [inline buzzfeed-style-quiz/src/script.js:28]: I removed this function and just placed the code directly in the init function

File: buzzfeed-style-quiz/src/style.less [hunk 4]
```diff
@@ -41,7 +41,7 @@ body {
     margin-right:10px;
   }
   
-  &.active {
+  &.chosen {
```
Comment (evanzhong) [inline buzzfeed-style-quiz/src/style.less:44]: Minor renaming, makes certain lines read more like English such as:

`let chosenAnswer = $(question).find('.quiz-answer.chosen').data('quiz-answer')`

**Comment by chasekap** [comment]
Awesome work!

**Pull request merged**